XenAdmin/Wizards/RollingUpgradeWizard/RollingUpgradeWizardPrecheckPage.cs

```diff
@@ -197,12 +197,15 @@ public override string HelpID
                 }
             }
 
-            //iSL (StorageLink) check
-            checks.Add(new KeyValuePair<string, List<Check>>(Messages.CHECKING_STORAGELINK_STATUS, new List<Check>()));
-            checkGroup = checks[checks.Count - 1].Value;
-            foreach (Host host in SelectedServers)
+            //iSL (StorageLink) check - CA-223486: only for pre-Creedence
+            if (SelectedServers.Any(h => !Helpers.CreedenceOrGreater(h)))
             {
-                checkGroup.Add(new HostHasUnsupportedStorageLinkSRCheck(host));
+                checks.Add(new KeyValuePair<string, List<Check>>(Messages.CHECKING_STORAGELINK_STATUS, new List<Check>()));
+                checkGroup = checks[checks.Count - 1].Value;
+                foreach (Host host in SelectedServers)
+                {
+                    checkGroup.Add(new HostHasUnsupportedStorageLinkSRCheck(host));

```

Comment:
I believe the check needs to be added on the individual servers too, because in case we are upgrading more than one pools the check will still run on all of them including the post-creedence ones.
I think the next check (clearwater) is similarly afflicted.
One can check the above by deselecting the hide successful checkbox at the bottom of the page so the checks on the individual servers are shown in detail.